doc/image_proc.rst

```diff
@@ -291,6 +291,29 @@ variations (``use_pixelflat``), the slit illumination profile
 These multiplicative corrections are all propagated to the image-processing
 error budget.  See :ref:`flat_fielding` for additional discussion.
 
+.. note::
+
+    Currently, to apply the slit-illumination and spectral response corrections,
+    you must also apply the pixel-to-pixel correction. I.e., in order to perform
+    *any* flat-field correction, ``use_pixelflat`` must be true.
+
+Continuum removal
+-----------------
+
+If multiple different arc frames are being combined, a smooth representation of
+the image can be subtracted off by setting the ``use_continuum`` variable to true.
+This variable should *only* be set to true if you intend to combine multiple arcs
+with different lamps.
+
+.. warning::
+
+    If you set this parameter to True, you should also set the following parameters:
+
+    calibrations arcframe process clip = False
+    calibrations arcframe process combine = 'mean'
+    calibrations tiltframe process clip = False
+    calibrations tiltframe process combine = 'mean'
```

Comment:
Thanks for these new docs!  Can we include a formatting statement and put these in the configobj format?  I.e., 
```
.. code-block:: ini

    [calibrations]
        [[arcframe]]
            [[[process]]]
                clip = False
                combine = mean
        [[tiltframe]]
            [[[process]]]
                clip = False
                combine = mean
```

Reply:
Thanks @kbwestfall for the code - I didn't know this was possible until now! I've added this to the docs